functions/slice/1.0/function.json

```diff
@@ -0,0 +1,47 @@
+{
+	"description": "Slices an array/collection from the given start value, for the given length.",
+	"label": "Slice",
+	"category": "Misc",
+	"icon": "CreateIcon",
+	"options": [
+		{
+			"meta": {
+				"type": "Text"
+			},
+			"name": "array",
+			"label": "Input array",
+			"info": "The original input array",
+			"advanced": false
+		},
+		{
+			"meta": {
+				"type": "Text"
+			},
+			"name": "start",
+			"label": "Start",
+			"info": "Start position (starts with 0)",
+			"advanced": false
+		},
+		{
+			"meta": {
+				"type": "Text"
+			},
+			"name": "length",
```

Comment:
Why not call this end and let the user decide how many items he / she wants from the array?